Cubical/Data/Nat/Order.agda

```diff
@@ -127,3 +168,66 @@ private
 <-wellfounded : WellFounded _<_
 <-wellfounded zero = acc λ _ → ⊥-elim ∘ ¬-<-zero
 <-wellfounded (suc n) = acc-suc (<-wellfounded n)
+
+module _
+    (b₀ : ℕ)
+    (P : ℕ → Set)
+    (base : ∀ n → n < suc b₀ → P n)
+    (step : ∀ n → P n → P (suc b₀ + n))
+  where
+  open WFI (<-wellfounded)
+
+  private
+    dichotomy : ∀ b n → (n < b) ⊎ (Σ[ m ∈ ℕ ] n ≡ b + m)
+    dichotomy b n
+      = case n ≟ b return (λ _ → (n < b) ⊎ (Σ[ m ∈ ℕ ] n ≡ b + m)) of λ
+      { (lt o) → inl o
+      ; (eq p) → inr (0 , p ∙ sym (+-zero b))
+      ; (gt (m , p)) → inr (suc m , sym p ∙ +-suc m b ∙ +-comm (suc m) b)
+      }
+
+    dichotomy<≡ : ∀ b n → (n<b : n < b) → dichotomy b n ≡ inl n<b
+    dichotomy<≡ b n n<b
+      = case dichotomy b n return (λ d → d ≡ inl n<b) of λ
+      { (inl x) → cong inl (m≤n-isProp x n<b)
+      ; (inr (m , p)) → ⊥-elim (<-asym n<b (m , sym (p ∙ +-comm b m)))
+      }
+
+    dichotomy+≡ : ∀ b m n → (p : n ≡ b + m) → dichotomy b n ≡ inr (m , p)
+    dichotomy+≡ b m n p
+      = case dichotomy b n return (λ d → d ≡ inr (m , p)) of λ
+      { (inl n<b) → ⊥-elim (<-asym n<b (m , +-comm m b ∙ sym p))
+      ; (inr (m' , q))
+      → cong inr (subtypeEquality
```

Comment:
I think this is where travis is tripping up

Reply:
Has `subTypeEquality` moved or been renamed?

Reply:
Oh, it has. Now it's `ΣProp≡`.